### DISCUSSION TITLE
xcode 13.3.1 swift compiler error

### DISCUSSION THREAD

**Discussion by ABNG** [flutter-stripe/flutter_stripe#712] [Q&A] [answered]
`Swift Compiler Error (Xcode): 'maestro' is only available in iOS 12.0 or newer.`

There are a lot of swift compiler errors like that and i can not build the app. i already tried
 - flutter clean
 - pod deintegrate
 - pod install

but nothing works.

> 

 - Device: iPhone 13 pro max
 - deployment target:  12.0
 - Package version: flutter_Stripe 2.5.0
 - Flutter version: 2.10.5

**Comment by jonasbark** [comment, marked as answer]
Check the Changelog:
> BREAKING: This library now supports iOS 12 and up, due to stripe-ios increasing the deployment target.
To upgrade your iOS deployment target to 12.0, you can either do so in Xcode under your Build Settings, or by modifying IPHONEOS_DEPLOYMENT_TARGET in your project.pbxproj directly. You will also need to update your Podfile to target :ios, '12.0'.
